Enterprise-CMCS/eAPD
# Tbolt/4491 medicaid business areas bug

Resolves #4491 

### Description
Fixes issue where certain Medicaid Business Areas were not being persisted in the backend/db. 

### Significant changes or possible side effects
* Minor change to the backend model

### Automated test cases written

| Given | When | Then | Type (jest, tap, cypress) |
| ----- | ---- | ---- | ------------------------- |
| A HITECH APD type selected | creating a new APD | all appropriate options are displayed correctly with the correct values | jest (snapshot) |
| A MMIS APD type selected | creating a new APD | all appropriate options are displayed correctly with the correct values | jest (snapshot) |

### Steps to manually verify this change
1. Login as a user who can create APDs
2. Create a new APD
3. Select MMIS as the type
4. Select *all 16* of the Medicaid Business Areas (including filling out Other)
5. Before hitting the Create APD button, open DevTools and select the Network Tab
6. Check that the response of the APD GET (by it's ID) has all 16 of the options in apd -> apdOverview -> medicaidBusinessAreas
<img width="1081" alt="Screenshot 2023-02-14 at 8 42 33 AM" src="https://user-images.githubusercontent.com/2279260/218755624-25582e9d-bff0-410f-8c78-9845e01c0d01.png">

### This pull request is ready to code review when

      tests, refactor
      screenreader, text scaling) OR an exemption is documented

### This pull request is ready to test when


### This pull request is ready to review when the QA has


### This pull request can be merged when
